cvat/apps/engine/utils.py

```diff
@@ -88,3 +91,8 @@ def rotate_image(image, angle):
     matrix[1, 2] += bound_h/2 - image_center[1]
     matrix = cv.warpAffine(image, matrix, (bound_w, bound_h))
     return matrix
+
+def md5_hash(frame):
+    if isinstance(frame, VideoFrame):
+        frame = frame.to_image()
+    return hashlib.md5(frame.tobytes()).hexdigest() # nosec
```

Comment:
@Marishka17 , could you please add a comment why you believe it is OK to disable the security warning?

Reply:
`md5` hash algorithm (like some [others](https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b303-md5)) bandit marks as unsafe, but in our specific case, there is no need to use a perfect cryptographic hash function. So I disable the security warnings for the PR to pass.